### PR TITLE
API-Docs Food

### PR DESCRIPTION
**Pull request by vhhieu95**
API-Docs Food

### CODE REVIEW

File: docs/api-references/food.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+#### Request Headers
+
+| Key | Value |
+|---|---|
+|Accept|application\json
+
+#### Response
+```json
+{
+    {
```
Comment (ghost) [inline docs/api-references/food.md]: Is not correct JSON structure.
You need to use like this:
```json
    "data": {
        "id": 123,
        "name": "Foooood",
        "category_id": 2,
        "price": 1000,
        "description": "",
        "image": "http://link-to-image.com/image_name.npg"
    },
```

**Review by ghost — COMMENTED** [review]
Please check my comment

File: docs/api-references/food.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            "id": 9,
+            "name": "Balistreri, Schaden and Swaniawski"
+        }
+    },
+    "susccess": true
+}
+```
+### `GET` List Foods
+```
+/foods
```
Comment (ghost) [inline docs/api-references/food.md]: If your route has prefix `/api/` please update your URI

**Review by ghost — COMMENTED** [review]
Please check my comment.

**Comment by at-hieuvo** [comment]
I updated, please check again. Thank you so much

**Comment by ghost** [comment]
LGTM

**Pull request merged**